.github/workflows/testDBChangelog.yml

```diff
@@ -0,0 +1,45 @@
+name: Test Liquibase rollbacks
+
+# Run this on pushes to any branch that changes a Liquibase changelog
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - "**"
+    paths:
+      - backend/src/main/resources/db/changelog/db.changelog-master.yaml
```

Comment:
Should we include the Metabase changelog in here too, since they're separated now?

Reply:
Maybe? I don't see that changelog changing all that much, and it's not our _data_, really, but I'm open to doing so. I didn't want to overcomplicate things.

Reply:
Fair enough! I also don't imagine we'll touch it too often, so I'm fine to leave it off.